src/daemon/Display.cpp

```diff
@@ -50,7 +54,30 @@
 #include "WaylandDisplayServer.h"
 
 
-namespace SDDM {
+namespace SDDM
+{
+    bool approveTty(const QString &desiredTty) {
```

Comment:
I don't like the name. "approve"  sounds like a verb, implying it does something active, also it's not a term logind or ioctl's are using

bool isTtyInUse(...)